test/internet/test-dns-ipv6.js

```diff
@@ -44,46 +40,49 @@ function checkWrap(req) {
 }
 
 TEST(function test_resolve6(done) {
-  var req = dns.resolve6('ipv6.google.com', function(err, ips) {
-    if (err) throw err;
+  const req = dns.resolve6('ipv6.google.com',
+    common.mustCall((err, ips) => {
```

Comment:
ok ... I will change it, was just confirming that this is the right format... was unsure because that code looks a bit odd to me idented like that...